packages/zowe-explorer/src/globals.ts

```diff
@@ -122,6 +122,12 @@ export const configurationDictionary: { [k: string]: string } = {
     "Zowe-Jobs-Persistent": SETTINGS_JOBS_HISTORY,
 };
 
+export enum EVENT_TYPES {
+    CREATE,
+    UPDATE,
+    DELETE,
+}
```

Comment:
I'm wondering if we can export this enum through the API so that extenders can differentiate between `EVENT_TYPES`? Otherwise, extenders will have to compare the enum values to hardcoded numbers, and if we make any changes to the `EVENT_TYPES` enum, extenders will have to update their logic accordingly.

This way, we can also update the function type in `registerProfileChangeCallback` to be more specific:
`callback: (eventType: EVENT_TYPES) => Promise<void>`